qa/rolling-upgrade/src/test/java/org/opensearch/knn/bwc/StatsIT.java

```diff
@@ -20,14 +20,16 @@
 public class StatsIT extends AbstractRollingUpgradeTestCase {
     private KNNStats knnStats = new KNNStats(KNN_STATS);
 
-    // Validate if all the KNN Stats metrics are returned
+    // Validate if all the KNN Stats metrics from old version are present in new version
```

Comment:
Need to change this test as it fails for newly added stats. We must ensure that all the old stats are not removed, but "equals" check also checks that there are no new stats added, which is incorrect